File: output/pygments/pygments_base16/base16-pastel-krwenholz.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#262626'
    base01 = '#a8ff60'
    base02 = '#ffffb6'
    base03 = '#4f4f4f'
    base04 = '#96cbfe'
    base05 = '#ffffff'
    base06 = '#ff73fd'
    base07 = '#eeeeee'
    base08 = '#ffb6b0'
    base09 = '#ff6c60'
    base0a = '#ffffcc'
    base0b = '#ceffac'
    base0c = '#dfdffe'
    base0d = '#b5dcff'
    base0e = '#ff9cfe'
    base0f = '#c6c5fe'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('pastel-krwenholz', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords

File: output/crosh/colors/base16-pastel-krwenholz.js
// Base16 Pastel
// Scheme: Kyle Wenholz (http://krwenholz.com)

var color_scheme = {
        'base00': '#262626',
        'base01': '#a8ff60',
        'base02': '#ffffb6',
        'base03': '#4f4f4f',
        'base04': '#96cbfe',
        'base05': '#ffffff',
        'base06': '#ff73fd',
        'base07': '#eeeeee',
        'base08': '#ffb6b0',
        'base09': '#ff6c60',
        'base0A': '#ffffcc',
        'base0B': '#ceffac',
        'base0C': '#dfdffe',
        'base0D': '#b5dcff',
        'base0E': '#ff9cfe',
        'base0F': '#c6c5fe',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(255, 255, 255, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);
